pom.xml

```diff
@@ -246,6 +246,40 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>exec-maven-plugin</artifactId>
```

Comment:
@VitaliiTytarenko-okta can we also add config to skip this plugin execution when needed? Ref: https://www.mojohaus.org/exec-maven-plugin/exec-mojo.html#skip

I am thinking abt cases where we need to skip this execution (in case of any issues with the spec from npm) and use the local spec (src/swagger/api.yaml) in this case. I think we should add this capability.

Reply:
of course